internal/core/workers/operation_execution_worker/operation_execution_worker.go

```diff
@@ -42,10 +44,13 @@ import (
 
 var _ workers.Worker = (*OperationExecutionWorker)(nil)
 
+const healthControllerExecutionIntervalConfigPath = "workers.operationExecution.healthControllerInterval"
+
 // OperationExecutionWorker is the service responsible for implementing the worker
 // responsibilities.
 type OperationExecutionWorker struct {
 	scheduler        *entities.Scheduler
+	config           config.Config
```

Comment:
@arthur29 usually we dont use the config interface directly in our domain structs, instead, we create a type for the config, receive it in the constructor, and use an external provider that knows how to build it. This will make the domain struct independent from the viper lib. you can see some examples here internal/service/config.go.